install_config/configuring_authentication.adoc

```diff
@@ -520,57 +746,40 @@ CLI) are sent a `WWW-Authenticate` challenge header for this provider.
 <3> When *true*, unauthenticated token requests from web clients (like the web
 console) are redirected to a login page backed by this provider.
 <4> Controls how mappings are established between this provider's identities and user objects,
-xref:mapping-identities-to-users[as described above].
-<5> List of attributes to use as the identity. First non-empty attribute is
-used. At least one attribute is required. If none of the listed attribute have a
-value, authentication fails.
-<6> List of attributes to use as the email address. First non-empty attribute is
-used.
-<7> List of attributes to use as the display name. First non-empty attribute is
-used.
-<8> List of attributes to use as the preferred user name when provisioning a
-user for this identity. First non-empty attribute is used.
-<9> Optional DN to use to bind during the search phase.
-<10> Optional password to use to bind during the search phase. This value may also be
-provided in an
-xref:../install_config/master_node_configuration.adoc#master-node-configuration-passwords-and-other-data[environment
-variable, external file, or encrypted file].
-<11> Certificate bundle to use to validate server certificates for the
-configured URL. If empty, system trusted roots are used. Only applies if
-*insecure: false*.
-<12> When *true*, no TLS connection is made to the server. When *false*,
-`ldaps://` URLs connect using TLS, and `ldap://` URLs are upgraded to TLS.
-<13> An RFC 2255 URL which specifies the LDAP host and search parameters to use,
-xref:ldap-url[as described above].
+xref:mapping-identities-to-users-table[as described above].
+<5> URL accepting credentials in Basic authentication headers.
+<6> Optional: Certificate bundle to use to validate server certificates for the
+configured URL.
+<7> Optional: Client certificate to present when making requests to the
+configured URL.
+<8> Key for the client certificate. Required if `*certFile*` is specified.
 ====
 
-[[BasicAuthPasswordIdentityProvider]]
-
-=== Basic Authentication (Remote)
-
-Set *BasicAuthPasswordIdentityProvider* in the `*identityProviders*` stanza to
-validate user names and passwords against a remote server using a
-server-to-server Basic authentication request. User names and passwords are
-validated against a remote URL that is protected by Basic authentication and
-returns JSON.
-
-A `401` response indicates failed authentication.
-
-A non-`200` status, or the presence of a non-empty "error" key, indicates an
```

Comment:
Appears after the example in a section called "Failed Authentication":
Failed Authentication
A 401 response indicates failed authentication.
A non-200 status, or the presence of a non-empty "error" key, indicates an error: